cogs/html.py

```diff
@@ -0,0 +1,37 @@
+import discord
+from discord.ext import commands
+import imgkit
+
+
+class Html(commands.Cog):
+    def __init__(self, bot):
+        self.bot = bot
+    
+    @commands.group(name="html")
+    async def html(self, ctx):
+        pass
+    
+    @commands.command(name="html", aliases=["html_to_img"])
+    async def html(self, *, code):
```

Comment:
if the code is long, then discord automatically turns it into a text file and attaches to the message, so in our case, the code can be defaulted to None, and if it is None, we can check for attachments. or it can be vice versa too, if there are no attachments, then we check for the argument passed.